rllib/agents/cql/tests/test_cql.py

```diff
@@ -38,31 +39,34 @@ def test_cql_compilation(self):
         data_file = os.path.join(rllib_dir, "tests/data/pendulum/small.json")
         print("data_file={} exists={}".format(data_file, os.path.isfile(data_file)))
 
-        config = {
-            "env": "Pendulum-v1",
-            "input": [data_file],
-            # In the files, we use here for testing, actions have already
-            # been normalized.
-            # This is usually the case when the file was generated by another
-            # RLlib algorithm (e.g. PPO or SAC).
-            "actions_in_input_normalized": False,
-            "clip_actions": True,
-            "train_batch_size": 2000,
-            "twin_q": True,
-            "learning_starts": 0,
-            "bc_iters": 2,  # 2 BC iters, 2 CQL iters.
-            "rollout_fragment_length": 1,
-            # Switch on off-policy evaluation.
-            "input_evaluation": ["is"],
-            "always_attach_evaluation_results": True,
-            "evaluation_interval": 2,
-            "evaluation_duration": 10,
-            "evaluation_config": {
-                "input": "sampler",
+        config = Trainer.merge_trainer_configs(
```

Comment:
Why do we need to merge here now? Doesn't Trainer do this automatically in `setup()`?